types/react-signature-canvas/index.d.ts

```diff
@@ -0,0 +1,38 @@
+// Type definitions for react-signature-canvas 1.0
+// Project: https://github.com/agilgur5/react-signature-canvas
+// Definitions by: Kamil Socha <https://github.com/ksocha>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as React from 'react';
+import SignaturePad from 'signature_pad';
+
+export interface SignatureCanvasProps extends SignaturePad.SignaturePadOptions {
```

Comment:
I'm against this as other type definitions rarely use this convention of prepending `I` to the interface name. I personally think it brings no value and I've even seen linter rules to prevent naming interfaces like this.

I will change the name to `ReactSignatureCanvasProps` though.